src/astro/databases/snowflake.py

```diff
@@ -445,9 +445,9 @@ def load_pandas_dataframe_to_table(
         :param if_exists: Strategy to be used in case the target table already exists.
         :param chunk_size: Specify the number of rows in each batch to be written at a time.
         """
-        self.create_table(target_table, dataframe=source_dataframe)
+        if if_exists == "replace":
```

Comment:
Does it do the check to see if the table already exists?

Reply:
create_table calls - 
1. create_table_using_schema_autodetection() - which calls prep_table() with if_exists="replace".
2. create_table_using_columns - I have added drop_table() to it.